nidirect_backlinks/nidirect_backlinks.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Contains nidirect_backlinks.module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function nidirect_backlinks_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.nidirect_backlinks':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Provides a local task on node content entities to outline what other content
+        refers to it from either:') . '</p>';
+      $output .= '<ul>';
+      $output .= '<li>' . t('Text (long / with summary)') . '</li>';
+      $output .= '<li>' . t('Link') . '</li>';
+      $output .= '<li>' . t('Entity reference') . '</li>';
+      $output .= '</ul>';
+      return $output;
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_node_insert().
+ */
+function nidirect_backlinks_node_insert(EntityInterface $entity) {
```

Comment:
Is there an event you can subscribe to for these, or can it only be done with hooks?